partials/tabs/projects.html

```diff
@@ -26,8 +26,6 @@
                 <br> </div>
               <div ng-show="project.collaborating_projects.length>0" class="project-overview-element clickable card-action"> <span ng-click="search(project)" class="pr-element-detail chip" ng-repeat="project in project.collaborating_projects">{{ project }}</span>
                 <br> </div>
-              <div ng-show="project.status=='crowded'" class="red white-text flag card-action center"> <span class="status-title">HIGH INVOLVEMENT</span>
-                <br> <span class="tiny">If you haven't put work into this, look out for other projects for better chances!</span> </div>
```

Comment:
I'm not finding a file called `projects.js`. Perhaps I'm misunderstanding. I could remove the `status: crowded` from the individual projects. Here's what I ran to try to locate the files you mentioned:

```
[~/projects]$ find ./ -name "projects.js"
[~/projects]$ grep -rnw './' -e "crowded"
./_projects/nested_languages.md:4:status: crowded
./_projects/communication_bridge.md:4:status: crowded
./_projects/optimize_caching.md:4:status: crowded
./_projects/bear_updates.md:4:crowded: true
./_projects/debug_profile.md:4:status: crowded
./partials/tabs/projects.html:29:              <div ng-show="project.status=='crowded'" class="red white-text flag card-action center"> <span class="status-title">HIGH INVOLVEMENT</span>
[~/projects]$
```